Library/Formula/srmio.rb

```diff
@@ -15,7 +15,7 @@ class Srmio < Formula
 
   def install
     if build.head?
-      system "chmod u+x genautomake.sh"
+      chmod 1, "genautomake.sh"
```

Comment:
@eirinikos Also (as this isn't documented obviously): it'd be good to use the Ruby 1.8 docs for `chmod` as we still have to support Ruby 1.8 for now, unfortunately :sob:: http://ruby-doc.org/stdlib-1.8.7/libdoc/fileutils/rdoc/FileUtils.html#method-c-chmod